cmd/deploy.go

```diff
@@ -59,162 +58,224 @@ type UpRequest struct {
 type Deployment struct {
 	*RemoteVPS
 	Repository *git.Repository
+	Auth       string
 }
 
-// deployCmd represents the deploy command
-var deployCmd = &cobra.Command{
-	Use:   "deploy [REMOTE] [COMMAND]",
-	Short: "Configure continuous deployment to the remote VPS instance specified",
-	Long: `Start or stop continuous deployment to the remote VPS instance specified.
-Run 'inertia remote status' beforehand to ensure your daemon is running.
-Requires:
+var deployUpCmd = &cobra.Command{
+	Use:   "up",
+	Short: "Bring project online on remote",
+	Long: `Bring project online on remote.
+	This will run 'docker-compose up --build'. Requires the Inertia daemon
+	to be active on your remote - do this by running 'inertia [REMOTE] init'`,
+	Run: func(cmd *cobra.Command, args []string) {
+		// Start the deployment
+		deployment, err := getDeployment()
+		if err != nil {
+			log.Fatal(err)
+		}
+		resp, err := deployment.Up()
+		if err != nil {
+			log.Fatal(err)
+		}
 
-1. A deploy key to be registered for the daemon with your GitHub repository.
-2. A webhook url to registered for the daemon with your GitHub repository.
+		defer resp.Body.Close()
+		body, err := ioutil.ReadAll(resp.Body)
+		if err != nil {
+			log.WithError(err)
+		}
+
+		switch resp.StatusCode {
+		case http.StatusCreated:
+			fmt.Printf("(Status code %d) Project up\n", resp.StatusCode)
+		case http.StatusForbidden:
+			fmt.Printf("(Status code %d) Bad auth: %s\n", resp.StatusCode, body)
+		default:
```

Comment:
Later we should handle the 412 response a bit more explicitly here (I think theres a 412 on this endpoint?). Not blocking.